ONSdigital/dp-frontend-dataset-controller
# Feature/one version per page

### What

- Add a versions list page
- Link to the versions list if there are previous versions on the edition
- Remove the local renderer client and use go-ns renderer client

### How to review

- Add a new version to existing dataset (can be done by finding an existing one in mongo and updating version fields)
- Visit edition landing page and verify there is a link to previous versions
- Verify link takes you to a version landing page

### Who

Anyone
